src/app/tests/suites/certification/Test_TC_S_2_5.yaml

```diff
@@ -250,7 +249,7 @@ tests:
           to 20000 (20s) and no extension field sets."
       PICS: S.S.C00.Rsp
       verification: |
-          ./chip-tool scenesmanagement add-scene 0x0001 0x01 20000 scene1 [] 1 1
+          scenesmanagement add-scene 0x0001 0x01 20000 scene1 [] 1 1
```

Comment:
This change will apply to all steps from Test step 5a onwards.

Reply:
Updated